forge/build.gradle

```diff
@@ -55,6 +55,7 @@ dependencies {
 	modImplementation("dev.latvian.mods:rhino-forge:${rootProject.rhino_version}") { transitive = false }
 	modImplementation("dev.latvian.mods:kubejs-forge:${rootProject.kubejs_version}") { transitive = false }
 
+	modLocalRuntime("me.shedaniel:RoughlyEnoughItems-forge:8.3.563")
```

Comment:
Should this still be in?

Reply:
Not strictly, but it doesn't hurt at all to have it - just a local runtime dependency in the dev instance.

Reply:
(probably also useful to keep around to verify we don't have a regression later on)